nixos/modules/services/misc/airsonic.nix

```diff
@@ -25,6 +25,14 @@ in {
         '';
       };
 
+      virtualHost = mkOption {
+        type = types.nullOr types.str;
+        default = "airsonic";
```

Comment:
The default should be `null`, right?

Reply:
tt-rss uses a default of `"tt-rss"`. Having said that, `null` is probably a better default choice.